monitoring/scripts/central-monitor-setup.ps1

```diff
@@ -0,0 +1,71 @@
+http_listener
+
+# Listener
+az network application-gateway http-listener create -g gpitfutures-monitor-rg-appgw --gateway-name gpitfutures-monitor-appgw --frontend-port gpitfutures-monitor-appgw-feporthttps -n rancher --frontend-ip gpitfutures-monitor-appgw-feip --host-name "rancher.dynamic.buyingcatalogue.digital.nhs.uk" --ssl-cert dyn-buying-catalogue-digital-nhs-uk
+
+# Backend Address Pool
+az network application-gateway address-pool create -g gpitfutures-monitor-rg-appgw --gateway-name gpitfutures-monitor-appgw -n rancher
+
+# Probe
+az network application-gateway probe create -g gpitfutures-monitor-rg-appgw --gateway-name gpitfutures-monitor-appgw -n rancher --protocol http --host "rancher.dynamic.buyingcatalogue.digital.nhs.uk" --path "/"
+
+# HTTP Setting
+az network application-gateway http-settings create -g gpitfutures-monitor-rg-appgw --gateway-name gpitfutures-monitor-appgw -n rancher --port 80 --protocol Http --cookie-based-affinity Disabled --timeout 20 --path "/" --probe "rancher"
+
+# Routing Rule
+az network application-gateway rule create -g gpitfutures-monitor-rg-appgw --gateway-name gpitfutures-monitor-appgw -n rancher --http-listener rancher --rule-type Basic --address-pool rancher --http-settings rancher
+
+# Kubernetes Setup
+az aks get-credentials -n gpitfutures-monitor-aks -g gpitfutures-monitor-rg-aks --admin
+
+helm repo add kubernetes-dashboard https://kubernetes.github.io/dashboard/
+helm install kubernetes-dashboard kubernetes-dashboard/kubernetes-dashboard --create-namespace --namespace kubernetes-dashboard
+
+kubectl create namespace ingress-appgw
+
+cd <path-to-repo>\platform-tf\monitoring\
+
+kubectl create -f "../ingress/deployment-rbac.yaml"
+
+#az identity list -g gpitfutures-monitor-rg-aks -o json > ./ingress/monitor-msi.txt
```

Comment:
Is this being reenabled in the future or can it be removed?

Reply:
Don't really see why we do it at all - I've removed the line. Thanks